core/block.ts

```diff
@@ -180,6 +180,13 @@ export class Block implements IASTNodeLocation, IDeletable {
   protected collapsed_ = false;
   protected outputShape_: number|null = null;
 
+  /**
+   * Is the current block currently in the process of being disposed?
+   *
+   * @internal
+   */
+  disposing = false;
```

Comment:
Now that you're no longer checking the property directly, this should probably be made private vs @internal.

Reply:
Done! Should be good for another look!